blacklist/src/commands/consult_email.py

```diff
@@ -0,0 +1,13 @@
+from src.commands.base_command import BaseCommannd
+from src.utils.blacklist_utils import get_blacklist_by_email
+
+
+class ConsultEmail(BaseCommannd):
```

Comment:
Seria chevere renombrarlo a algo como: is email bannered...

Reply:
Se ajusta nombre : [6259a9c](https://github.com/4304-2024-15-devops-barichara-team/2024-blacklist/pull/6/commits/6259a9c4be62b89c639245b5c1fa4971c2122fbb)